docs/setting-up/server/docker.md

```diff
@@ -367,6 +367,18 @@ Use the following Docker container environment variables (with `-e var=value`) t
 | `PMM_PUBLIC_ADDRESS`                                            | External IP address or the DNS name on which PMM server is running.
 | `PMM_DISABLE_BUILTIN_POSTGRES`                                  | Disable built-in PostgreSQL database.
 
+The following variables are also supported but values passed are not verified by PMM. If any other variable is found, it will be considered invalid and the server won't start.
+
+| Variable                                                        | Description
+| --------------------------------------------------------------- | ------------------------------------------------------
+| `_`, `HOME`, `HOSTNAME`, `LANG`, `PATH`, `PWD`, `SHLVL`, `TERM` | Default environment variables.
+| `GF_*`                                                          | Grafana's environment variables.
```

Comment:
```suggestion
| `GF_*`                                                          | Grafana's environment variables.
| `VM_*`                                                         | Victoria metrics' environment variables.
```

Reply:
@rasika-chivate here I think we should add links to Grafana and Victoria metrics documentation about their environment variables.

Reply:
Sure